README.md

```diff
@@ -5,42 +5,81 @@
 
 <img src="icon.png" width="20%"/>
 
-Picoleto is the gentleman that will validate your Spanish identity documents.
+Picoleto is the gentleman that will validate your Spanish documents:
+
+-Identity
+
+    -NIF
+    -NIE
+    -Passport
+
+-Phones
+
+    -MobileLine
+    -LandLine
+
+-ZipCodes
+
```

Comment:
The indented lines appears as code blocks, what does not looks really good... can you check it out?

Reply:
Fixed!